src/rules/noImplicitDependenciesRule.ts

```diff
@@ -60,7 +60,7 @@ export class Rule extends Lint.Rules.AbstractRule {
     /* tslint:enable:object-literal-sort-keys */
 
     public static FAILURE_STRING_FACTORY(module: string) {
-        return `Module '${module}' is not listed as dependency in package.json`;
+        return `Module '${module}' is not listed as dependency in relevant section(s) of package.json`;
```

Comment:
@newren what would be _really cool_ is if it could tell you what the relevant section _is_, which we can probably determine from options. you'd need to add an arg to this function. how do you feel about taking a stab at that??

Reply:
Don't currently have the time to look at this like I did three months ago; maybe eventually...